web/client/plugins/Identify.jsx

```diff
@@ -104,7 +104,7 @@ const selector = createStructuredSelector({
  */
 const identifyIndex = compose(
     connect(
-        createSelector(indexSelector, isLoadedResponseSelector, (state) => state.browser && state.browser.mobile, (index, loaded, isMobile) => ({ index, loaded, isMobile })),
+        createSelector(indexSelector, isLoadedResponseSelector, (state) => state.browser && state.browser.mobile, (state) => state.localConfig, (index, loaded, isMobile, localConfig) => ({ index, loaded, isMobile, localConfig })),
```

Comment:
please restore to previous selector based on the previous comment about plugin configuration

Reply:
Do we need this state.localConfig after the changes? @tonym3nsa